src/Client/Client.php

```diff
@@ -55,13 +56,24 @@ public function createRequest(Request $request)
      *
      * @param \Assertis\Http\Request\Request $request
```

Comment:
It can and can't. It can't because there is already Request imported from different namespace. I think this is better way to handling that.